recipes/area-detector-handlers/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "area-detector-handlers" %}
+{% set version = "0.0.8" %}
+{% set sha256 = "676cec048d2d41136d758fcdfe94a53aca551d2d3c267daf33c4a112a5662f18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - dask
```

Comment:
Upstream also lists pandas? Looks like the only reason that's not failing is because the test import below doesn't actually trigger much in the way of importing. Might want to add an import of `handlers` as well.